### ISSUE TITLE
Unclear instructions

### ISSUE THREAD

**Issue by thillcafex** [microsoftgraph/microsoft-graph-docs#7760]
&gt; Give consent to opt in to Microsoft Graph data connect through the Microsoft 365 Admin Portal, on the Services & add-ins page.

I have no idea where this is.  It's not under Settings &gt; Add-ins

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0302f7b1-e754-fd0a-4a8a-057ed5deaa9a
* Version Independent ID: f819a46b-23d4-65c4-fc12-5b558b81511a
* Content: [Get started with Microsoft Graph data connect - Microsoft Graph](https://docs.microsoft.com/en-us/graph/data-connect-get-started)
* Content Source: [concepts/data-connect-get-started.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/concepts/data-connect-get-started.md)
* Product: **data-connect**
* Technology: **microsoft-graph**
* GitHub Login: @ajacks-msft
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by thillcafex** [comment]
You have to switch back to the 'old' admin center, then follow the instructions here:
https://github.com/microsoftgraph/msgraph-training-dataconnect/blob/master/Lab.md

**Comment by chr1sn3al** [comment]
You can access the Premission via https://admin.microsoft.com/Adminportal/Home?source=applauncher#/Settings/Services/:/Settings/L1/O365DataPlan

Seach for MIcrosoft Graph data connect..   You will need a distribution list to set for approvals

**Comment by fmms** [comment]
Yes the documentation is indeed very hard to follow here.

One has to go to https://admin.microsoft.com/Adminportal/Home

Then one has to click on "Show all"
![image](https://user-images.githubusercontent.com/908734/94343929-06512e80-001c-11eb-95ec-19552641e87a.png)

Next step is:
![image](https://user-images.githubusercontent.com/908734/94343945-24b72a00-001c-11eb-8072-02f9775df2d0.png)

hope this helps and these instruction can be added to the official docs.

**Comment by ajacks-msft** [comment]
This is now fixed. Thank you for reporting.
